src/lib/holocene/button.svelte

```diff
@@ -131,7 +131,7 @@
   }
 
   .link {
-    @apply border-0 bg-none p-0 font-primary text-sm underline shadow-none;
+    @apply border-0 bg-none p-0 font-primary text-sm text-blue-700 leading-5 font-semibold hover:underline shadow-none;
```

Comment:
Wondering if we want another `Button` variant for all the `Cancel` use cases that use `link`.

|Before | After|
|--|--|
|![Screenshot 2023-07-18 at 3 02 59 PM](https://github.com/temporalio/ui/assets/15069288/e98526e0-162e-474e-bd8b-19851476ef62)|![Screenshot 2023-07-18 at 3 04 43 PM](https://github.com/temporalio/ui/assets/15069288/50d4c4a2-8e97-4eed-b3b9-e6b655a1a476)|

Reply:
This is the css for the link variant in the designs, so it's supposed to be new styles for link button. I looked and that data encoder settings UI is the only other spot we have a link button. I say we roll with it there too (I like it more and don't want to add another variant).

Reply:
Fair enough. Just wanted to make sure it was a change we wanted in Cloud too since we use the `link` variant for this pattern all over the place ⬇️ 

![Screenshot 2023-07-19 at 9 24 42 AM](https://github.com/temporalio/ui/assets/15069288/1e563ea0-e0c0-47dd-8127-2cd3e89b3e23)